test/JDBC/expected/BABEL-2844.out

```diff
@@ -104,6 +124,11 @@ Query Text: select 1
 Result  (cost=0.00..0.01 rows=1 width=4)
 ~~END~~
 
+~~START~~
+text
+Babelfish T-SQL Batch Parsing Time: 0.083 ms
```

Comment:
How can we keep this test not flaky when we showed the real exec time for parsing ?

Reply:
It will be ignore during file diff comparison in JDBC framework